src/Main.elm

```diff
@@ -95,24 +90,42 @@ update msg model =
         ( GoogleApiSignOut, NotAuthed ) ->
             ( model, Cmd.none )
 
-        ( MessageIdsLoaded result, Authed state ) ->
-            ( { model
-                | state = Authed { state | messageIds = RemoteData.fromResult result }
-              }
-            , Cmd.none
-            )
+        ( ThreadsLoaded result, Authed state ) ->
+            let
+                ( threads, cmd ) =
+                    case RemoteData.fromResult result of
+                        RemoteData.NotAsked ->
+                            ( RemoteData.NotAsked, Cmd.none )
+
+                        RemoteData.Loading ->
+                            ( RemoteData.Loading, Cmd.none )
+
+                        RemoteData.Failure error ->
+                            ( RemoteData.Failure error, Cmd.none )
+
+                        RemoteData.Success { threads } ->
+                            ( List.map
+                                (\thread ->
+                                    ({ thread = thread
+                                     , messages = RemoteData.Loading
+                                     }
+                                    )
+                                )
+                                threads
+                                |> RemoteData.Success
+                            , Cmd.batch
+                                []
```

Comment:
@gyzerok Would it be like super bad thing to try and initialize the `View.Thread` here ?